.gitignore

```diff
@@ -26,4 +26,3 @@ project.xcworkspace
 #
 node_modules/
 npm-debug.log
-dist/
```

Comment:
Please leave the dist folder in the gitignore, it is generated and should not be commited